target/linux/mediatek/files-5.4/drivers/mtd/mtk-snand/Kconfig

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (C) 2020 MediaTek Inc. All rights reserved.
+# Author: Weijie Gao <weijie.gao@mediatek.com>
+#
+# SPDX-License-Identifier: GPL-2.0
```

Comment:
SPDX license identifier must be first line in the file (unless there is a shebang, then it must be second line).

Please adjust this for _all_ files in the various commits here.